core/src/main/kotlin/api/AdminRoute.kt

```diff
@@ -64,6 +68,13 @@ fun Route.admin() = route("admin") {
     route("users") {
         val userService by inject<UserService>()
 
+        get(getUsers) {
+            requireSuperuser()
+
+            val users = userService.getUsers()
+            call.respond(users)
```

Comment:
Endpoints should only return classes from the `:api:v1:model` module.

Reply:
@mnonnenmacher can you explain how to avoid this from detekt? Do I have to rewrite the mapping to map `Set<User>`(from model) -> into a `Set<User>` (from API) instead of mapping the classes directly?
```
> Task :core:detektMain FAILED
/home/runner/work/ort-server/ort-server/core/src/main/kotlin/api/AdminRoute.kt:75:52: Name shadowed: implicit lambda parameter 'it' [NoNameShadowing]
```